app/javascript/readingList/readingList.jsx

```diff
@@ -13,32 +12,49 @@ import {
 import { ItemListItem } from './components/ItemListItem';
 import { ItemListItemArchiveButton } from './components/ItemListItemArchiveButton';
```

Comment:
We should at some point look into components names: `ItemListItem` is probably not very descriptive...